test/language_command_interface_tests.cpp

```diff
@@ -0,0 +1,161 @@
+#include <gtest/gtest.h>
+
+#include "isobus/isobus/can_NAME_filter.hpp"
+#include "isobus/isobus/can_internal_control_function.hpp"
+#include "isobus/isobus/can_managed_message.hpp"
+#include "isobus/isobus/can_parameter_group_number_request_protocol.hpp"
+#include "isobus/isobus/can_partnered_control_function.hpp"
+#include "isobus/isobus/can_stack_logger.hpp"
+#include "isobus/isobus/isobus_language_command_interface.hpp"
+
+using namespace isobus;
+
+// A log sink for the CAN stack
+class CustomLogger : public isobus::CANStackLogger
+{
+public:
+	void sink_CAN_stack_log(CANStackLogger::LoggingLevel, const std::string &) override
+	{
+		// Do nothing
+	}
+};
+
+static CustomLogger logger;
```

Comment:
Generally I agree, but every time I use the logger it increases the number of uncovered conditions, probably related to exception-only paths, so in addition to testing the logger in its own file, I think we might want to run gcovr with `--exclude-throw-branches`

Reply:
I also think we should add that in. It would've been useful to include these 'throw-branches' if we were throwing exceptions ourselves, but we aren't. So it's fine by me